README.md

```diff
@@ -48,6 +51,34 @@ pacman -S base-devel git ncurses lib32-zlib gawk time unzip perl-xml-libxml \
  flex wget gettext quilt python2 openssl
 ```
 
+
+### Build-System-Structure  
+
+Where can I change something? What does these files do? 
+
+```
+:: Settings
+
+config.mk             - OpenWRT git repository, revision, select what package lists to build
+feeds.conf            - OpenWRT feeds.conf that is used for the package feeds during build
```

Comment:
The whole file moved to the _modules_ file

Reply:
adjusted in new commit